src/main/resources/org/jenkinsci/plugins/schedulebuild/ScheduleBuildAction/index.jelly

```diff
@@ -5,8 +5,8 @@
         <l:main-panel>
             <h1>${%Project} ${it.owner.name}</h1>
             <f:form name="schedule" action="next" method="post">
-                <f:entry title="${%BuildOn} (mm/dd/yy):" field="date">
-                    <f:textbox id="schedule-build-input" value="${it.defaultDate}" checkUrl="${rootURL}/${it.owner.url}schedule/checkDate" checkDependsOn=""/>
+                <f:entry title="${%BuildOn} (${it.dateTimeFormatting}):" help="${rootURL}/plugin/schedule-build/help/help-date.html" description="Timezone: ${it.timeZone}">
```

Comment:
Thanks very much for adding the help.  Unfortunately, when I click the help icon from within the job page, it displays "File not found" rather than showing the help page.

It looks like this when I click the help icon from a Freestyle job that takes no parameters:

![help-file-not-found](https://github.com/jenkinsci/schedule-build-plugin/assets/156685/000952dc-a586-4a1d-bf41-4d02c7f589d7)

I like the help very much but am also OK if you decide to remove the help rather than find the location to place the help in the plugiin layout

Reply:
fixed (for the help url the use of ${rootURL} is not needed)